README.rst

```diff
@@ -92,8 +92,10 @@ How?
 
 1. Get a personal access token with permission to manage your *gists*
    from `GitHub settings <https://github.com/settings/tokens>`__
-2. Set an environment variable called ``GETGIST_TOKEN`` with your
-   personal access token
+2. Set an environment variable called ``GETGIST_TOKEN`` with your personal access token
+
+
+>This `article <https://www.serverlab.ca/tutorials/linux/administration-linux/how-to-set-environment-variables-in-linux/>`__ might help you create an environment >variable in a Unix-based operational system with Bash, but feel free to search alternatives for other systems and shells.
```

Comment:
Hi @kmmelcher. Viewing the [redered version of the `README.rst`](https://github.com/cuducos/getgist/blob/f04504b8625a2b48f09d589e4c79a5ca954dbb80/README.rst) it looks like there are two `>` characters seem like they shouldn't be there. Also, there's an unneeded extra blank line.

What do yo think about this minor edits?

```suggestion

This `article <https://www.serverlab.ca/tutorials/linux/administration-linux/how-to-set-environment-variables-in-linux/>`__ might help you create an environment variable in a Unix-based operational system with Bash, but feel free to search alternatives for other systems and shells.
```